controllers/osbuild_controller.go

```diff
@@ -46,9 +46,9 @@ const (
 	EdgeContainerJobStillRunningMsg = "Edge-container job is still running"
 
 	// OSBuildConditionTypes values
-	ContainerBuildDone    = "ContainerBuildDone"
```

Comment:
In my next PR I'm changing those consts to be in v1alpha1 as I saw it was done in [kubevirt](https://github.com/kubevirt/kubevirt/blob/f3ddc0c65b54f4d1d0ad9d583f330cdff7385dd3/staging/src/kubevirt.io/api/core/v1/types.go#L1860)... 
In their case - the values are in the Upper case

Reply:
Currently it's broken and builds cannot be triggered. So, let's merge this PR and change it later after testing